docs/docs/howto/encrypted-storage.md

```diff
@@ -1,4 +1,18 @@
-# Encrypted volume mount
+# Set up encrypted persistent storage
+
+This section provides a tutorial on how to configure encrypted persistent storage for your application.
+
+## Applicability
+
+This step is recommended for any Contrast deployment that stores sensitive data persistently.
+
+## Prerequisites
+
+1. [Set up your cluster](./cluster-setup/aks.md)
+
+## How-to
+
+<!-- TODO: I think this has to be changed to really make clear what steps are needed for encrypted persistant storage. Otherwise it is again more or less a tutorial for deploying an already configured Contrast-protected deplyoment. -->
```

Comment:
Fully agree: we should focus on the mysql app and its extra annotation here, marking runtime as prereq and only linking to docs for `generate` etc